.github/workflows/broken-link-checker.yml

```diff
@@ -18,4 +18,7 @@ jobs:
           cache-dependency-path: 'yarn.lock'
           node-version-file: '.nvmrc'
       - run: yarn install
-      - run: yarn blc ${{ github.event.inputs.url }} --ro
+      - name: Check broken link
+        env:
+          url: ${{ github.event.inputs.url || 'https://es-toolkit.slash.page' }}
```

Comment:
```suggestion
          url: ${{ github.event.inputs.url || 'https://es-hangul.slash.page' }}
```

Reply:
@okinawaa I have updated it now, sorry for the confusion.